Update service creation for Promscale steps (#1697)
timescale/docs: _partials/_cloud-create-service-promscale.md

1.  Sign in to the [Timescale Cloud portal][timescale-cloud]
1.  Click `Create service`.
1.  Click `Advanced configuration` and set the `Region`, `Compute`, and `Disk size`
    for the service. With 1&nbsp;CPU, 4&nbsp;GB and 250&nbsp;GB you can ingest and
    store up to 10,000&nbsp;metric samples per second with the default 3-month retention. 
    Check our [resource recommendation guide][promscale-resource-recomm] for configuring the compute and 
    disk size based on your ingestion rate and retention.

>     <highlight type="note">
>     Make sure the Promscale Connector and the service in Timescale cloud are
>     on the same region. Ideally we recommend the latency between the Promscale 
>     Connector and Database to be less than 100&nbsp;ms. 
>     </highlight>

Reviewer: ```suggestion

    <highlight type="note">
    Make sure the Promscale Connector and the service in Timescale Cloud are
    in the same region. Ideally, the latency between the Promscale 
    Connector and database should be less than 100&nbsp;ms. 
    </highlight>
    
```

1.  Sign in to the [Timescale Cloud portal][timescale-cloud]
1.  Click `Create service`.
1.  Click `Advanced configuration` and set the `Region`, `Compute`, and `Disk size`
    for the service. With 1&nbsp;CPU, 4&nbsp;GB and 250&nbsp;GB you can ingest and
    store up to 10,000&nbsp;metric samples per second with the default 3-month retention. 

>     Check our [resource recommendation guide][promscale-resource-recomm] for configuring the compute and 

Reviewer: ```suggestion
    Check the [resource recommendation guide][promscale-resource-recomm] for configuring the compute and 
```
